libc/test/src/math/smoke/CanonicalizeTest.h

```diff
@@ -135,44 +135,38 @@ class CanonicalizeTest : public LIBC_NAMESPACE::testing::Test {
 
       // Exponent   |       Significand      | Meaning
       //            |   Bit 63   | Bits 62-0 |
-      // All Other  |   Zero     | Anything  | Unnormal, Value =
-      //  Values    |            |           | (−1)**s × m × 2**−16382
+      // All Other  |   Zero     | Anything  | Unnormal, Value = SNaN
+      //  Values    |            |           |
 
-      FPBits test5_1(UInt128(0x0000000000000001));
```

Comment:
Sorry I just remember that we do have `_u128` literal for testing: https://github.com/llvm/llvm-project/blob/main/libc/test/src/__support/str_to_long_double_test.cpp#L24

Do you mind updating this test file to use it?  They will need to depend on `libc.src.__support.integer_literals` target.

Thanks,

Reply:
that test is updated to this now: 

```
FPBits test5_1((UInt128(0x0040) << 64) + UInt128(0x0000000000000001));
```

Should I update all the tests?

Reply:
Yes, all the `UInt128` literals in this test file.

Reply:
done.